configs/nr-otel-collector.yaml

```diff
@@ -0,0 +1,57 @@
+# TODO We need to decide what default config we want here. For now this is a c&p from otel releases.
+
+extensions:
+  health_check:
+  pprof:
+    endpoint: 0.0.0.0:1777
+  zpages:
+    endpoint: 0.0.0.0:55679
+
+receivers:
+  otlp:
```

Comment:
Should we add the default configuration of the hostmetrics receiver?

Reply:
as we discussed we need to decide on the default config. Let's leave it as it is for now (with the TODO https://github.com/newrelic/opentelemetry-collector-releases/pull/2/files#diff-fc66cb839ba9e9a13bbe1d2c9ba7aa65b3b9c917c7563381511ff4f15057fb09R1)